website/static/js/share/index.js

```diff
@@ -87,6 +92,14 @@ ShareApp.controller = function() {
 
     });
 
+    m.request({
+        method: 'GET',
+        url: '/api/v1/share/documents/' // TODO where will the postgres API live??
+    }).then(function(data) {
+        self.vm.rawNormedLoaded(true);
+    }, function(err) {
+        // blank, but needed to handle missing rawNormed API
```

Comment:
Yes, an error is expected here - for now, until we get the SHARE Postgres api up and online, this should throw an error. I know @lyndsysimon looked a bit into handling it, and made a conclusion to leave it. But I'll check in with him and fix it as such!